components/cards/cards.css

```diff
@@ -0,0 +1,84 @@
+@import url("https://pro.fontawesome.com/releases/v5.10.0/css/all.css");
+
+body {
+  font-family: var(--nunito);
+}
+
+.showcase-div {
+  display: flex;
+  flex-direction: column;
+  padding: var(--spacing-xl);
+  gap: var(--spacing-xl);
+}
+
+.card {
+  margin-top: 2rem;
+  flex-shrink: 0;
+  position: relative;
+  height: 25rem;
+  width: 18rem;
+  box-shadow: var(--shadow-subtle);
+}
+
+.img-size {
+  height: 18rem;
+  width: 18rem;
+}
+
+.pad-topleft {
+  padding-left: var(--spacing-2xs);
+  padding-top: var(--spacing-3xs);
+}
+
+.btns {
+  width: max-content;
+  margin-left: auto;
+  padding: 0.2rem;
+}
+
+.btn {
+  border: none;
+  background-color: transparent;
+}
+
+.card-badge {
+  background-color: var(--blue-pr);
+  position: absolute;
+  padding: 0 0.2rem;
+  top: var(--spacing-sm);
+}
+
+.cross {
+  height: 2rem;
+  color: var(--white);
+  position: absolute;
+  font-size: var(--fs-xl);
+  right: var(--spacing-sm);
+  top: var(--spacing-sm);
+}
+
+.text-overlay {
+  position: absolute;
+  bottom: 8rem;
+  font-weight: var(--fw-bold);
+  padding-right: var(--spacing-xs);
+  background-color: rgba(255, 255, 255, 0.151);
+}
+
+.text-only {
+  height: 12rem;
+}
+
+.hrz-card {
+  display: flex;
+  align-items: center;
```

Comment:
if flex uses more in the code you can make utility class for it

Reply:
Thanks i'll consider it.